src/fsharp/FSharp.Core/list.fsi

```diff
@@ -297,6 +553,16 @@ namespace Microsoft.FSharp.Collections
         /// <param name="list">The input list.</param>
         ///
         /// <returns>A list containing only the elements that satisfy the predicate.</returns>
+        /// 
+        /// <example id="filter-example-1">
+        /// <code lang="fsharp">
+        /// let input = [1, "Luke"; 2, "Kirk"; 3, "Kenobi"; 4, "Spock"]
+        /// let isComingFromStarTrek = fst >> isEven
```

Comment:
Please write without using `>>` as we try not to use constructs more "complicated" (later in the learning sequence) than those being taught